building/build-debs/homeworld-admin-tools/src/iso.py

```diff
@@ -29,11 +29,11 @@ def regen_cdpack(source_iso, dest_cdpack):
         subprocess.check_call(["tar", "-czf", dest_cdpack, "-C", d, os.path.basename(cddir)])
 
 
-def add_password_to_log(password):
+def add_password_to_log(password, now):
```

Comment:
Minor, but "creation_time" would be a better variable name here.

Reply:
Will fix.